recipe_scrapers/bakels.py

```diff
@@ -0,0 +1,80 @@
+# mypy: allow-untyped-defs
+
+from ._abstract import AbstractScraper
+from ._grouping_utils import group_ingredients
+
+
+class Bakels(AbstractScraper):
+    @classmethod
+    def host(cls):
+        return "bakels.com.au"
+
+    def author(self):
+        return "Australian Bakels"
+
+    def title(self):
+        return self.soup.find("h1").get_text()
+
+    def image(self):
+        return self.schema.image()
+
+    def ingredients(self):
+        div = self.soup.find("div", id="tab-ingredients_1")
+        if not div:
+            return
+
+        results = []
+        groups = div.find_all("div", {"class": "row"})
+        for group in groups:
+            ingredients = group.find_all(class_="text-xs-left")
+            for ingredient in ingredients:
+                text = ingredient.get_text(strip=True)
+                if text == "Ingredient" or text.startswith("Group"):
+                    continue  # Ensures these headings are not included as ingredients
+                results.append(text)
+        return results
+
+    def ingredient_groups(self):
+        groups = group_ingredients(
+            self.ingredients(),
+            self.soup,
+            ".group-label",
+            "#tab-ingredients_1 .text-xs-left:not(:-soup-contains('Ingredient')):not(:-soup-contains('Group'))",
```

Comment:
Nice!  It's the filtering of the `text-xs-left` elements that I wasn't able to figure out.  Smart :)

Is the `-soup-contains` condition some functionality specific-to / provided-by BeautifulSoup?

Reply:
Yes it is. It's used to filter elements based on their text content -> `:-soup-contains('text')`